attunement.lic

```diff
@@ -9,21 +9,41 @@ class Attunement
   include DRCA
   include DRCT
 
+  LUNAR_PERCEIVE_COMMANDS = ['mana', 'moons', 'planets']
+
   def initialize
     settings = get_settings
     @stationary_skills_only = settings.crossing_training_stationary_skills_only
     @hometown = settings.hometown
     @attunement_rooms = settings.attunement_rooms
-    @start_timer = Time.now
+    @exp_timer = settings.exp_timers['Attunement'] || 90
+    @exp_max_threshold = settings.crossing_training_max_threshold
```

Comment:
I'm not sure how I feel about this... this setting has been specific to the `crossing-training` script up until now...

Reply:
I'm open to either. Can you (quickly) poll the user base and see if there's a strong opinion either way?

Reply:
Without me digging through the whole works, if someone wants to give me a question or two to ask, i can put up a survey

Reply:
Thanks @Hiinky! 

We're wanting to know if people want to use `exp_timers` to limit how long this script will run (like a time upper limit) and `crossing_training_max_threshold` as the max learning rate limit.

Those are two existing settings used by crossing-trainer script. That script calls this attunement script. Does it make sense to re-use those crossing-training settings here or should we introduce new settings?

Reply:
@rpherbig @Hiinky @BinuDR 

To simplify things, I'm going with Option 2 unless ya'll think otherwise. I'll introduce `attunement_target_increment` setting and if set means how many mind states you want the script to increase your Attunement skill before stopping. Players can set a small value to run the script for short time, or a large value to ensure you get to a high mind state.

`crossing_training_max_threshold` used by crossing-training script is still an upper threshold. The training script won't call any of the sub-scripts like attunement if your mind state is at or above that threshold.

`exp_timers` will stay exclusive to crossing-training script. Rather than use a timer, the attunement script will rely on `crossing_training_max_threshold` and/or `attunement_target_increment`.

### Examples

**Example 1**
```yaml
crossing_training_max_threshold: 30
attunement_target_increment: 6
```
Player's attunement is at 20. The player runs attunement script and it checks if skill(20) >= threshold(30). It's not so we proceed to next step to determine how long to run the script. The threshold(30) - skill(20) is 10 mind states to go. However, the target increment is 6 so once the skill(20) has increased to at least 26 then the script will stop.

**Example 2**
```yaml
crossing_training_max_threshold: 30
attunement_target_increment: 6
```
Player's attunement is at 28. The player runs attunement script and it checks if skill(28) >= threshold(30). It's not so we proceed to next step to determine how long to run the script. The threshold(30) - skill(28) is 2 mind states to go. Even though the target increment is 6 the script will stop once the skill is at 30 (28+2), not 34 (28+6).